vic/drivers/classic/src/vic_classic.c

```diff
@@ -223,15 +237,17 @@ main(int   argc,
                 /**************************************************
                    Compute cell physics for 1 timestep
                 **************************************************/
+                timer_start(&cell_timer);
```

Comment:
The accumulation is done in `put_data` below.

Remember, I have two timers going here.

1.  measures the execution time of `vic_run` for individual grid cells, 
2. measures the aggregate run time for the init/run/final steps of the model.